netbox/project-static/styles/custom/_misc.scss

```diff
@@ -28,3 +28,9 @@ span.color-label {
   visibility: hidden;
   opacity: 0;
 }
+
+// Override column filter form dropdown
```

Comment:
What is this overriding? It looks like a custom class; the comment should explain its purpose.